resources/docs/TUTORIAL_8_MODEL_OPTIMIZATION.md

```diff
@@ -0,0 +1,156 @@
+# Tutorial 8: Model Tuning
+
+This is part 8 of the tutorial, in which we look into how we can improve the quality of our model by selecting
+the right set of model and hyper parameters.
+
+## Selecting Hyper Parameters
+
+Flair includes a wrapper for the well-known hyper parameter selection tool
+[hyperopt](https://github.com/hyperopt/hyperopt).
+
+First you need to load your corpus. If you want to load the [AGNews corpus](https://www.di.unipi.it/~gulli/AG_corpus_of_news_articles.html)
+used in the following example, you first need to download it and convert it into the correct format. Please
+check [tutorial 6](/resources/docs/TUTORIAL_6_CORPUS.md) for more details.
+```python
+from flair.datasets import TREC_6
+
+# load your corpus
+corpus = TREC_6()
```

Comment:
The tutorial is a bit counterintuitive since it defines a text classification corpus and then word embeddings. The fact that there's an LSTM used is buried in the model initialization and not explained.

Perhaps switch to a sequence labeling example? `WNUT_17` for instance could be used as example corpus, and for sequence labeling the current implementation allows some interesting parameter searches.

Reply:
I agree - makes sense to give precedence to sequence labelling in Flair.

As per your suggestion I added an example to the top that uses `WNUT_17 ` and documents the use of `SequenceTaggerParamSelector `

However, I also kept the example that shows how to use `TextClassifierParamSelector`. I feel like that if a feature isn't documented, people are unlikely to dig through the source and the value of the feature will be diminished.

So what I did was I created two sections (`Selecting hyperparameters for sequence labelling` and `Selecting hyperparameters for text classification`) and documented both use-cases accordingly.